sfa_dash/blueprints/base.py

```diff
@@ -105,15 +105,11 @@ def parse_start_end_from_querystring(self):
         """
         start_arg = request.args.get('start')
         end_arg = request.args.get('end')
-        # if start and end are not present, set them to an invalid datetime
-        # to trigger value error
-        if not start_arg:
-            start_arg = "x"
-        if not end_arg:
-            end_arg = "x"
```

Comment:
define `utc_now = pd.Timestamp.utcnow()` up here so you're using a consistent value throughout the function.